contrib/opencensus-ext-azure/opencensus/ext/azure/metrics_exporter/__init__.py

```diff
@@ -239,11 +245,57 @@ def _transmit_without_retry(self, envelopes):
                 text,
             )
 
+    def enable_standard_metrics(self):
+        # Sets up stats related objects to begin
+        # recording standard metrics
+        stats_ = stats.stats
+        view_manager = stats_.view_manager
+        stats_recorder = stats_.stats_recorder
+
+        # Standard metrics uses a separate instance of MeasurementMap
+        # from regular metrics but still shares the same underlying
+        # map data in the context. This way, the processing during
+        # record is handled through a separate data structure but
+        # the storage only uses one single data structure
+        # Uniqueness should be based off the name of the view and
+        # measure. A warning message in the logs will be shown
+        # if there are duplicate names
+        self.standard_metrics_map = stats_recorder.new_measurement_map()
+
+        free_memory_measure = measure_module.MeasureInt("Free memory",
+            "Amount of free memory in bytes",
+            "bytes")
+        self.standard_metrics_measure_map[StandardMetricType.FREE_MEMORY] = free_memory_measure
+        free_memory_view = view_module.View(StandardMetricType.FREE_MEMORY,
+            "Amount of free memory in bytes",
+            [],
+            free_memory_measure,
+            aggregation_module.LastValueAggregation())
+        view_manager.register_view(free_memory_view)
+
+    def record_standard_metrics(self):
+        # Function called periodically to record standard metrics
+        vmem = psutil.virtual_memory()
+        free_memory = vmem.free
```

Comment:
Do we need "Available Memory" for Azure Monitor, or "Free Memory"?
It seems what Azure Monitor has is Available Memory.

Reply:
You are correct. I was following Node SDK which uses os.freemem() but apparently that returns the available memory. I will change this.